wire-ios-transport/Source/Public/ZMUpdateEvent.swift

```diff
@@ -69,9 +69,9 @@ import WireUtilities
     case userPropertiesDelete = 33
     case teamCreate = 23
     case teamDelete = 24
-    // removed: teamUpdate = 25 [WPB-4552]: The event is no longer sent, clients must fetch team metadata (e.g. name, icon) every 24h 
-    // removed: teamMemberJoin = 26 [WPB-4538]: no need to handle "team.member-join" in clients
-    case teamMemberLeave = 27 // [WPB-4538]: "team.member-leave" is only required for backwards compatibility
+    case teamUpdate = 25
+    // removed: teamMemberJoin = 26 [WPB-4538]: "team.member-join" should not be handled anymore
+    case teamMemberLeave = 27
```

Comment:
Why is the `teamUpdate` introduced if it should be removed (PR description says so)?

Reply:
My mistake, thanks for pointing it out.